OWNERS

```diff
@@ -1,20 +1,17 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-  - chauhankaranraj
-  - michaelclifford
-  - pacospace
-  - tumido
+  - Shreyanand
+  - suppathak
+  - erikerlandson
+  - codificat
 
 reviewers:
   - aakankshaduggal
-  - chauhankaranraj
   - hemajv
-  - isabelizimm
```

Comment:
Hi @Shreyanand . Not sure if Isabel , Margaret, and karan are still active in the organization repo?

Reply:
I don't think they are and so removed them.